src/RadioBroadcast.php

```
@@ -43,4 +53,59 @@ public function getName()
+     * re-render cannot drift apart.
+     */
+    public function toArray(): array
+    {
+        return [
+            'name' => $this->decode($this->getName()),
+            'start' => $this->start->timestamp,
+            'end' => $this->end->timestamp,
+            'start_time' => $this->start->format('H:i'),
+            'end_time' => $this->end->format('H:i'),
+            'label' => $this->getDayLabel(),
+            'show' => $this->show ? $this->serializeShow() : null,
```

PHPStan (2.2.7)

[warning] Ternary operator condition is always true.
• Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your %configurationFile%.
• If Streekomroep\RadioBroadcast::getName() is impure, add `@phpstan-impure` PHPDoc tag above its declaration. Learn more: https://phpstan.org/blog/remembering-and-forgetting-returned-values
• If Streekomroep\RadioBroadcast::decode() is impure, add `@phpstan-impure` PHPDoc tag above its declaration. Learn more: https://phpstan.org/blog/remembering-and-forgetting-returned-values
• If Streekomroep\RadioBroadcast::getDayLabel() is impure, add `@phpstan-impure` PHPDoc tag above its declaration. Learn more: https://phpstan.org/blog/remembering-and-forgetting-returned-values

(ternary.alwaysTrue)